cal/api/views.py

```diff
@@ -241,14 +242,18 @@ class TagDetailEvents(generics.ListAPIView):
     serializer_class = GEventSerializer
 
     def get_queryset(self):
+        query_calendar = self.request.query_params.get('calendar_ids')
```

Comment:
name of intermediate variable should probably be something like `calendar_ids_str` to maintain consistency with the rest of the app (you can see another example in my PR). Imo `query_calendar` is not very descriptive.